capstone/src/main/java/com/google/sps/servlets/clubs/EditClubServlet.java

```diff
@@ -90,6 +99,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
   }
 
+  private void updateMemberRequestList(
+      String nameToUpdate, HttpServletRequest request, Entity clubEntity, boolean isExclusive) {
```

Comment:
nit: it's better for method parameters to not be boolean flags used for forking logic (eg: isExclusive). Instead, this should be checked when the method is called above, and depending on the boolean, then calling the method or not.